ods_ci/tests/Resources/Page/Operators/OpenShiftPipelines.resource

```diff
@@ -17,45 +17,18 @@ ${REDHAT_OPENSHIFT_PIPELINES_YAML}      ods_ci/tests/Resources/Files/redhat-open
 Install Red Hat OpenShift Pipelines
-    [Documentation]    Install Red Hat OpenShift Pipelines operator depending on
-    ...     OpenShift version, platform (AWS/GCP/...) and RHODS flavor (Managed/Self-Managed) with the objective of
-    ...     testing all the operator versions without having to test all possible combinations:
-    ...     - OpenShift 4.11:                      OpenShift Pipelines 1.9
-    ...     - OpenShift 4.12, 4.13 (AWS and GCP):  OpenShift Pipelines 1.10 / 1.11 (depending on platform and flavor)
-    ...     - OpenShift >= 4.12 (OpenStack):       OpenShift Pipelines 1.11
-    ...     - OpenShift 4.14:                      OpenShift Pipelines 1.12
-    ...     Note: the complete operator compatiblity matrix is published at
-    ...      https://docs.openshift.com/container-platform/4.13/cicd/pipelines/op-release-notes.html
-    ${oc_version}=    Get OpenShift Version
-    ${is_self_managed}=    Is RHODS Self-Managed
-    ${platform}=    Fetch Cluster Platform Type
-    IF    "4.11" in "${oc_version}"
-        ${openshift_pipelines_version}=    Set Variable    pipelines-1.9
-    ELSE IF    "4.12" in "${oc_version}" or "4.13" in "${oc_version}"
-        IF    ${is_self_managed} == ${TRUE} and "${platform}" == "AWS"
-            ${openshift_pipelines_version}=    Set Variable    pipelines-1.10
-        ELSE IF    ${is_self_managed} == ${FALSE} and "${platform}" == "AWS"
-            ${openshift_pipelines_version}=    Set Variable    pipelines-1.11
-        ELSE IF    ${is_self_managed} == ${TRUE} and "${platform}" == "GCP"
-            ${openshift_pipelines_version}=    Set Variable    pipelines-1.11
-        ELSE IF    ${is_self_managed} == ${FALSE} and "${platform}" == "GCP"
-            ${openshift_pipelines_version}=    Set Variable    pipelines-1.10
-        ELSE
-            ${openshift_pipelines_version}=    Set Variable    pipelines-1.12
-        END
-    ELSE IF    "4.14" in "${oc_version}"
-        ${openshift_pipelines_version}=    Set Variable    pipelines-1.12
-    ELSE
-        ${openshift_pipelines_version}=    Set Variable    pipelines-1.12
-    END
-    Log    message=Installing OpenShift Pipelines operator: ${openshift_pipelines_version} (${platform} ${oc_version} Self-Managed: ${is_self_managed})
-    ...    console=True
-
+    [Documentation]    Install the latest Red Hat OpenShift Pipelines Operator by default. A different version
+    ...    can be installed by setting OPENSHIFT_PIPELINES_VERSION in test-variables.yml .
+    ...    Example: if the value of OPENSHIFT_PIPELINES_VERSION is pipelines-1.12 will install the latest 1.12 version
+    ...    that currently is 1.12.2
     ${return_code}=    Run And Return Rc
-    ...    sed -i "s,channel: .*,channel: ${openshift_pipelines_version},g" ${EXECDIR}/${REDHAT_OPENSHIFT_PIPELINES_YAML}
+    ...    sed -i "s,channel: .*,channel: ${OPENSHIFT_PIPELINES_VERSION},g" ${EXECDIR}/${REDHAT_OPENSHIFT_PIPELINES_YAML}
```

Comment:
`OPENSHIFT_PIPELINES_VERSION` is confusing as in fact it is `OPENSHIFT_PIPELINES_CHANNEL`

Reply:
Do you prefer OPENSHIFT_PIPELINES_CHANNEL instead of OPENSHIFT_PIPELINES_VERSION ?

Reply:
yes

Reply:
Done